en/config/autotune.md

```diff
@@ -0,0 +1,223 @@
+   Takeoff carefully and manually test that the vehicle is stable.
+7. If any strong oscillations occur, land immediately and follow the instructions in the [Troubleshooting](#troubleshooting) section below.
+
+VTOL vehicles are tuned Tuning is the same as multicopter in MC mode and fixed-wing in FW mode.
+You should repeat the tuning process for each case.
+
+## Parameters
+
+By default MC vehicles land before parameters are applied, while FW vehicles apply the parameters in-air and then test that the controllers work properly.
+This behaviour can be configured using the [MC_AT_APPLY](../advanced_config/parameter_reference.md#MC_AT_APPLY) and [FW_AT_APPLY](../advanced_config/parameter_reference.md#FW_AT_APPLY) parameters respectively:
+
+* `0`: the gains are not applied.
+  This is used for testing purposes if the user wants to inspect results of the auto-tuning algorithm without using them directly.
+* `1`: apply the gains after disarm (default for multirotors).
+  The operator can then test the new tuning while taking-off carefully.
+* `2`: apply immediately (default for fixed-fings).
+  The new tuning is applied, disturbances are sent to the controller and the stability is monitored during the next 4 seconds.
+  If the control loop is unstable, the control gains are immediately reverted back to their previous value.
+  If the test passes, the pilot can then use the new tuning.
+
+
+Fixed wing vehicles (only) can tune which axes are tuned using the [FW_AT_AXES](../advanced_config/parameter_reference.md#FW_AT_AXES) bitmask parameter:
+
+* bit `0`: roll (default)
+* bit `1`: pitch (default)
+* bit `2`: yaw
+
+
+### Troubleshooting
+
+#### The drone oscillates when performing the testing maneuvers prior to the multirotor auto-tuning
+
+* slow oscillations (1 oscillation per second or slower): this often occurs on large platforms and means that the attitude loop is too fast compared to the rate loop -> decrease `MC_[ROLL|PITCH]_P` by steps of 1.0.
+* fast oscillations (more than 1 oscillation per second): this is because the gain of the rate loop is too high -> decrease `MC_[ROLL|PITCH|YAW]RATE_K` by steps of 0.1
+
+#### The auto-tuning sequence fails
+
+If the drone was not moving enough during auto-tuning, the system identification algorithm might have issues to find the correct coefficients.
+Increase the `MC/FW_AT_SYS_ID` by steps of 1 and trigger the auto-tune again.
+
+#### The drone oscillates after auto-tuning
+
+Due to effects not included in the mathematical model such as delays, saturation, slew-rate, airframe flexibility, the loop gain can be too high.
+To fix this, reduce the `MC_[ROLL|PITCH|YAW]RATE_K` parameters by steps of 0.1 until the oscillation stops.
+If a slow oscillation appears, reduce `FW_[RPY]R_P` by steps of 0.5.
+
+
+### FAQ
+
+#### What frames types are supported?
+
+Autotuning is enabled for multicopter, fixed wing, and VTOL vehicles.
```

Comment:
```suggestion
Autotuning is enabled for multicopter, fixed wing, and hybrid VTOL fixed wing vehicles.
```

Reply:
Is this how we're referring to VTOLs now days? I think it is good to be precise, given that VTOL is not. But FWIW we don't make the distinction elsewhere and VTOL == hybrid VTOL fixed wing vehicles.